demo/demo2.js

```diff
@@ -0,0 +1,96 @@
+const { command, processText, Format, CommandResult } = require('../src');
+
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const grid = [
+  [ 'Lush Forest', 'Tropical Beach' ],
+  [ 'Chilly Tundra', 'Dry Desert' ],
```

Comment:
Did you steal these from somewhere, or did you come up with 'Chilly Tundra'? I love it.